claim/gql_mutations.py

```diff
@@ -258,6 +264,10 @@ def create_file(date, claim_id, document):
     return file_path
 
 
+# dynamic import from attachment_strategies directory, this will be marked as unresolved reference by most linters
+attachment_strategies_dict = {"claimdoc": handle_claimdoc_attachment}
```

Comment:
here just import the dict from __init__

Reply:
What's the point of that change if it has to be harcoded anyway? It will show error still I think and it is a temporary solution since you wanted me to move to signals.

Reply:
Ok, moved it anyway, the problem is still the same, because PyCharm analyses this statically, not dynamically, but it is in a different file.